src/local-fs.ts

```diff
@@ -72,15 +72,19 @@ export class LocalFileSystem implements FileSystem {
                             type: 'fileCreated',
                             fullPath: relPath.split(path.sep).join(pathSeparator),
                             newContent: content
-                        })));
+                        }))
+                        .catch(() => {})
```

Comment:
@tyv I can see 2 ways of handling it:
 - add a hook ( a second emitter, `onError()` or any other flavor) that is not part of the abstract `FileSystem` API but is still public on `local-fs`. That will supply a user application a way to be notified on errors.
 - supply a hard-coded strategy for errors (i.e simply log to console, or pessimistically re-read the underlying file system and look for inconsistencies)